modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,59 +66,99 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+
 [discrete]
 == Redfish APIs
 
-Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infrastructure.
+The installation program calls several Redfish API endpoints on the BMC when using installer-provisioned infrastructure on bare metal.
 
 [IMPORTANT]
 ====
-You need to ensure that your BMC supports all of the redfish APIs before installation.
+Ensure that your BMC supports all of the Redfish APIs before installation.
 ====
 
-List of redfish APIs::
-* Power on
+.Procedure
+
+. Set the BMC user name by running the following command:
++
+[source,terminal]
+----
+$ export USER=<username> <1>
+----
+<1> Replace `<username>` with the name of the BMC user.
+
+. Set the BMC password by running the following command:
++
+[source,terminal]
+----
+$ export PASS=<password> <1>
+----
+<1> Replace `<password>` with the password of the BMC user.
```

Comment:
I'm not srue these are needed. Customers should be familiar with use-repalced values. If we had a large example, calling out replaceable values would be cumbersome if loads existed.

Reply:
(Not from the docs team, but FWIW I agree, its unnecessary and increases doc length)